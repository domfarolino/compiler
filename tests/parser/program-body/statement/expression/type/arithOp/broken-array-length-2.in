program dominic is

  integer result[0: 10];
  integer left[0: 9];
  integer right[0: 9];

begin

  result := left + right;

end program.
