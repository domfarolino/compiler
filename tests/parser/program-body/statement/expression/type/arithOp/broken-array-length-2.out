[1;31mLine 9: Assignment statement target and expression arrays must be of the same length
Line 9: Error parsing program body
[0m